tests/common/test_event.py

```diff
@@ -50,31 +53,43 @@ def setUp(self):
-            'EventPid': os.getpid(),
-            'TaskName': threading.current_thread().getName(),
-            'KeywordName': '',
-            'IsInternal': False,
+            CommonTelemetryEventSchema.GAVersion: CURRENT_AGENT,
+            CommonTelemetryEventSchema.ContainerId: AgentGlobals.get_container_id(),
+            CommonTelemetryEventSchema.EventTid: threading.current_thread().ident,
+            CommonTelemetryEventSchema.EventPid: os.getpid(),
+            CommonTelemetryEventSchema.TaskName: threading.current_thread().getName(),
+            CommonTelemetryEventSchema.KeywordName: '',
             # common parameters computed from the OS platform
-            'OSVersion': EventLoggerTools.get_expected_os_version(),
-            'ExecutionMode': AGENT_EXECUTION_MODE,
-            'RAM': int(osutil.get_total_mem()),
-            'Processors': osutil.get_processor_cores(),
+            CommonTelemetryEventSchema.OSVersion: EventLoggerTools.get_expected_os_version(),
+            CommonTelemetryEventSchema.ExecutionMode: AGENT_EXECUTION_MODE,
+            CommonTelemetryEventSchema.RAM: int(osutil.get_total_mem()),
+            CommonTelemetryEventSchema.Processors: osutil.get_processor_cores(),
             # common parameters from the goal state
-            'VMName': 'MachineRole_IN_0',
-            'TenantName': 'db00a7755a5e4e8a8fe4b19bc3b330c3',
-            'RoleName': 'MachineRole',
-            'RoleInstanceName': 'MachineRole_IN_0',
+            CommonTelemetryEventSchema.TenantName: 'db00a7755a5e4e8a8fe4b19bc3b330c3',
+            CommonTelemetryEventSchema.RoleName: 'MachineRole',
+            CommonTelemetryEventSchema.RoleInstanceName: 'MachineRole_IN_0',
             # common parameters
-            'Location': EventLoggerTools.mock_imds_data['location'],
-            'SubscriptionId': EventLoggerTools.mock_imds_data['subscriptionId'],
-            'ResourceGroupName': EventLoggerTools.mock_imds_data['resourceGroupName'],
-            'VMId': EventLoggerTools.mock_imds_data['vmId'],
-            'ImageOrigin': EventLoggerTools.mock_imds_data['image_origin'],
+            CommonTelemetryEventSchema.Location: EventLoggerTools.mock_imds_data['location'],
+            CommonTelemetryEventSchema.SubscriptionId: EventLoggerTools.mock_imds_data['subscriptionId'],
+            CommonTelemetryEventSchema.ResourceGroupName: EventLoggerTools.mock_imds_data['resourceGroupName'],
+            CommonTelemetryEventSchema.VMId: EventLoggerTools.mock_imds_data['vmId'],
+            CommonTelemetryEventSchema.ImageOrigin: EventLoggerTools.mock_imds_data['image_origin'],
         }
 
+        self.expected_extension_events_params = {
+            GuestAgentExtensionEventsSchema.IsInternal: False,
+            GuestAgentExtensionEventsSchema.ExtensionType: ""
+        }
+
+    @staticmethod
+    def _is_guest_extension_event(event):
+        return event.eventId in (None, TELEMETRY_EVENT_EVENT_ID) and \
```

Comment:
Why is None a valid option here?

Reply:
A lot of the tests were using None value by default so I just kept that behavior for tests. Earlier we used to always add those 2 columns for all tables so this check was not needed before. I can check if removing this check breaks a lot of tests. If it does I would rather keep the None check here rather than fixing the tests as they dont add any logical value to our testing.

Reply:
I think this comment is similar to the other one I posted on the check against None... let's do try to remove it if not needed and not keep it there just for the sake of the tests.

Reply:
(if needed for functionality of the agent of course it can stay there)

Reply:
Done, thankfully wasnt that big a pain that I thought it would be